expat/CMakeLists.txt

```diff
@@ -412,7 +413,7 @@ if(EXPAT_BUILD_FUZZERS)
             "-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++.")
     endif()
 
-    string(FIND "${CMAKE_EXE_LINKER_FLAGS}" "-fsanitize" sanitizer_present)
+    string(FIND "${CMAKE_C_FLAGS}" "-fsanitize" sanitizer_present)
```

Comment:
What's the idea behind the change of variable — could you elaborate?

Reply:
The idea is that, with the introduction of ossbuild, we can no longer assume that `CMAKE_EXE_LINKER_FLAGS` will always be `-fsanitize`. For the afl build, it will actually be equal to something like `/usr/lib/libFuzzingEngine.a` whereas for the libfuzzer build it will be `-fsanitize=fuzzer`

However, one thing that is common to both afl and libfuzzer builds is that CFLAGS will contain a `-fsanitize=` string. So, in summary the switch from linker flags to cflags generalizes what this check was intended for to a scenario where the build can be targeted for either the afl or libfuzzer backends.